lib/core-integration/src/Test/Integration/Scenario/CLI/Network.hs

```diff
@@ -30,16 +31,94 @@ import Test.Hspec
+
+    it "CLI_NETWORK - cardano-wallet network parameters - \
+       \proper parameter epoch parameter" $ \ctx -> do
+        params1 <- getNetworkParamsViaCLIok ctx "latest"
+        params1 `shouldBe` expectedBlockchainParams
+
+        params2 <- getNetworkParamsViaCLIok ctx "0"
+        params2 `shouldBe` expectedBlockchainParams
+
+        info <- getNetworkInfoViaCLI ctx
+        let (EpochNo currentEpoch) = currentEpochNum info
+        params3 <- getNetworkParamsViaCLIok ctx (show currentEpoch)
+        params3 `shouldBe` expectedBlockchainParams
+
+        let prevEpoch = if (currentEpoch > 0)
+                then currentEpoch - 1
+                else currentEpoch
+        params4 <- getNetworkParamsViaCLIok ctx (show prevEpoch)
+        params4 `shouldBe` expectedBlockchainParams
+
+    it "CLI_NETWORK - cardano-wallet network parameters - \
+       \improper parameter epoch parameter" $ \ctx -> do
+
+        let wrong1 = "earlier"
+        params1 <- getNetworkParamsViaCLIwrong ctx "earlier"
+        params1 `shouldContain` (errMsg400MalformedEpoch wrong1)
+
+        let wrong2 = "1.1"
+        params2 <- getNetworkParamsViaCLIwrong ctx wrong2
+        params2 `shouldContain` (errMsg400MalformedEpoch wrong2)
+
+        info <- getNetworkInfoViaCLI ctx
+        let (EpochNo currentEpoch) = currentEpochNum info
+        let wrong3 = show $ currentEpoch + 10
+        params3 <- getNetworkParamsViaCLIwrong ctx wrong3
+        params3 `shouldContain` (errMsg404NoEpochNo wrong3)
+  where
+      getNetworkParamsViaCLIok
+          :: Context t
+          -> String
+          -> IO ApiNetworkParameters
+      getNetworkParamsViaCLIok ctx epoch = do
+          let port = show (ctx ^. typed @(Port "wallet"))
+          (Exit c, Stderr e, Stdout o) <- cardanoWalletCLI @t
+              ["network", "parameters", "--port", port, epoch ]
+          c `shouldBe` ExitSuccess
+          e `shouldContain` cmdOk
+          expectValidJSON (Proxy @ApiNetworkParameters) o
+
+      getNetworkParamsViaCLIwrong
+          :: Context t
+          -> String
+          -> IO String
+      getNetworkParamsViaCLIwrong ctx epoch = do
+          let port = show (ctx ^. typed @(Port "wallet"))
+          (Exit c, Stderr e, Stdout o) <- cardanoWalletCLI @t
+              ["network", "parameters", "--port", port, epoch ]
+          c `shouldBe` (ExitFailure 1)
+          o `shouldBe` ""
+          pure e
+
+      getNetworkInfoViaCLI
+          :: Context t
+          -> IO ApiNetworkInformation
+      getNetworkInfoViaCLI ctx = do
+          let port = show (ctx ^. typed @(Port "wallet"))
+          (Exit c, Stderr e, Stdout o) <- cardanoWalletCLI @t
+              ["network", "information", "--port", port ]
+          c `shouldBe` ExitSuccess
+          e `shouldContain` cmdOk
+          expectValidJSON (Proxy @ApiNetworkInformation) o
+
+      currentEpochNum :: ApiNetworkInformation -> EpochNo
+      currentEpochNum netInfo =
```

Comment:
```suggestion
      currentEpochNo netInfo =
```
For consistency with the name of the type `EpochNo`.

Reply:
done